index.html

```diff
@@ -0,0 +1,102 @@
+        </div>
+    </div>    
+        <!--  main 이미지 -->
+    <main>
+        <div class="main-content">
+            <div class="main-section">
+                <img src="img/Img_home_01.png">
+                <div class="main-description">
+                    <p class="description-1">Hot item</p>
+                    <h2 class="description-2">인기 상품을<br>
+                        확인해 보세요</h2>
+                    <p class="description-3">가장 HOT한 중고거래 물품을<br>
+                        판다 마켓에서 확인해 보세요</p>
+                </div>        
+            </div>            
+        </div>
+        
+        <div class="main-content">
+            <div class="main-section-2">
+                <img src="img/Img_home_02.png">
+                <div class="main-description-2">
+                    <p class="description-1">Search</p>
+                    <h2 class="description-2">구매를 원하는<br>
+                        상품을 검색하세요</h2>
+                    <p class="description-3">구매하고 싶은 물품은 검색해서<br>
+                        쉽게 찾아보세요</p>
+                </div>        
+            </div>
+        </div>
+
+        <div class="main-content">
+            <div class="main-section-3">
+                <img src="img/Img_home_03.png">
+                <div class="main-description-3">
+                    <p class="description-1">Register</p>
+                    <h2 class="description-2">판매를 원하는<br>
+                        상품을 등록하세요</h2>
+                    <p class="description-3">어떤 물건이든 판매하고 싶은 상품을<br>
+                        쉽게 등록하세요</p>
+                </div>        
+            </div>
+        </div>
+    </main>
+    <!-- footer 이미지 -->
+    <footer>
+        <div class="container">
+            <div class="bottom-banner">
+                <h2 class="section">믿을 수 있는<br>
+                    판다마켓 중고 거래</h2>
+                <img class="banner-img" src="img/Img_home_bottom.png">
+            </div>
+        </div>
+        <div class="footer-bg">
+            <p class="fodter-small">©codeit - 2024</p>
+            <div class="footer-buttons">
+                <a class="footer-button" href="/privacy.html">Privacy Polick</a>
+                <a class="footer-button" href="/faq.html">FAQ</a>
+            </div>
+            <div class="social-icon">
+                <a class="icon" href="https://www.facebook.com/" target="_blank">
+                    <img src="icon img/ic_facebook.png">
+                </a>
+                <a class="icon" href="https://x.com/" target="_blank">
+                    <img src="icon img/ic_twitter.png">
+                </a>
+                <a class="icon" href="https://www.youtube.com/" target="_blank">
+                    <img src="icon img/ic_youtube.png">
+                </a>
+                <a class="icon" href="https://www.instagram.com/" target="_blank">
+                    <img src="icon img/ic_instagram.png">
+                </a>
+            </div>
```

Comment:
💊 제안
이 영역은 로고들의 모음이라고 할 수 잇습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
            <ul class="social-icon">
              <li>
                <a class="icon" href="https://www.facebook.com/" target="_blank">
                    <img src="icon img/ic_facebook.png" alt="facebook 로고">
                </a>
              </li>
            </ul>
```